buildspec.yml

```diff
@@ -2,16 +2,16 @@ version: 0.2
 
 env:
   variables:
-    FRAMEWORK_VERSION: '1.10.2'
-    EIA_FRAMEWORK_VERSION: '1.3.1'
+    FRAMEWORK_VERSIONS: '2.0.0 2.0.1'
+    # EIA_FRAMEWORK_VERSION: '1.3.1'
```

Comment:
Let's remove related comments if we're not running EIA.

Reply:
We have a cleanup step planned in a subsequent PR where we will remove unnecessary files and code. I will remove comments related to EIA at that time.